src/core/variable.jl

```diff
@@ -351,6 +351,40 @@ function variable_reactive_dcline_flow(pm::GenericPowerModel; bounded = true)
     return pm.var[:q_dc]
 end
 
+"variable: `t_shift[l,i,j]` for `(l,i,j)` in `arcs`"
+function variable_phase_shift(pm::GenericPowerModel; bounded = true)

```

Comment:
This class of variables does not need the bounded flag.  This is only necessary for variables that are used in Power Flow studies.